### PR TITLE
Make introduction/overview more action-oriented

### PR DESCRIPTION
**Pull request by joelhans**
This PR makes the introduction (now renamed _Overview_) more focused on helping readers self-select and take the action most valuable to them—particularly the quickstart.

Here is a summary of the changes:

- Added grids/boxes for the most important actions/functions within Timeplus to the _Overview_ doc
  - This includes two new JSX components (`/src/components/...`) and their CSS
- Moved the **High performance storage**, **Powerful analytic engine**, and **End to end analytic platform** content to a separate _Why Timeplus?_ page, which also contains the _Showcase_ doc
- Added the Poppins font for brand consistency from the Timeplus website to the documentation 
- Small visual improvements

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,20 +1,54 @@
...
-Please check the [Showcases](showcases) page for a comprehensive list of our key use cases and capabilities.
-
+---
+title: Overview
+---
+
+import Quickstart from '@site/src/components/Quickstart'
+import Grid, { GridItem } from '@site/src/components/Grid'
+
+Timeplus is a streaming-first data analytics platform. It provides powerful end-to-end capabilities, leveraging the [Proton engine](proton), to help teams process streaming and historical data quickly and intuitively, accessible for organizations of all sizes and industries. It enables data engineers and platform engineers to unlock streaming data value using SQL.
```
Comment (jovezhong) [inline docs/index.md:8]: do you think we can mention open source? "leveraging the open source streaming database Proton"

File: docs/index.md [hunk 2]
```diff
@@ -1,20 +1,54 @@
...
+---
+
+import Quickstart from '@site/src/components/Quickstart'
+import Grid, { GridItem } from '@site/src/components/Grid'
+
+Timeplus is a streaming-first data analytics platform. It provides powerful end-to-end capabilities, leveraging the [Proton engine](proton), to help teams process streaming and historical data quickly and intuitively, accessible for organizations of all sizes and industries. It enables data engineers and platform engineers to unlock streaming data value using SQL. 
+
+The Timeplus console allows for easy connection to diverse data sources (such as Apache Kafka, Confluent Cloud, Redpanda, CSV file upload, and more), explore streaming patterns via SQL queries, send real-time insights and alerts to other systems or individuals, and create dashboards and visualizations.
+
+Still curious about [why to use Timeplus](why-timeplus)? Check out our [showcase](showcases) for use cases, sourced from
```
Comment (jovezhong) [inline docs/index.md:12]: not sure showcase or showcases

File: docs/index.md [hunk 3]
```diff
@@ -1,20 +1,54 @@
...
+    <h3>Write SQL queries &rarr;</h3>
+    <p>Create long-running queries using functions like transformations, joins, aggregation, windowed processing, substreams, and more.</p>
+  </GridItem>
+  <GridItem href="/viz">
+    <h3>Visualize data &rarr;</h3>
+    <p>View real-time results for any query, create custom dashboards to tell stories about your data, or integrate with external BI systems.</p>
+  </GridItem>
+</Grid>
+
+## Core concept and functions
```
Comment (jovezhong) [inline docs/index.md:39]: should we use "concepts"?

**Review by jovezhong — COMMENTED** [review]
Very cool, thank Joel! No outstanding issue, just some minor details to confirm.

**Review by jovezhong — CHANGES_REQUESTED** [review]
Confirmed with our designer @heyitswei 
Let’s keep Inter for the docs, because the main website to Inter as well

File: src/css/custom.css [hunk 4]
```diff
@@ -30,7 +31,7 @@ html[data-theme='dark'] {
   --ifm-color-primary-lightest: #EDB0CF;
   --ifm-code-font-size: 95%;
   --ifm-background-color: #2c2c2c;
-  --ifm-font-family-base: 'Inter';
+  --ifm-font-family-base: 'Poppins', 'Inter';
```
[the remark this answers is not in the document]
Comment (joelhans) [reply src/css/custom.css]: Done.

File: docs/index.md (hunk 3 above)
Comment (joelhans) [reply docs/index.md]: Yes—corrected!

File: docs/index.md (hunk 1 above)
Comment (joelhans) [reply docs/index.md]: Done.

**Review by jovezhong — APPROVED** [review]

**Pull request merged**